utils/src/userInput/showQuickPick.ts

```diff
@@ -149,12 +150,11 @@ function getRecentlyUsedKey(options: types.IAzureQuickPickOptions): string | und
     return recentlyUsedKey;
 }
 
-async function initializePicks<TPick extends types.IAzureQuickPickItem<unknown>>(picks: TPick[] | Promise<TPick[]>, options: types.IAzureQuickPickOptions, groups: QuickPickGroup[], recentlyUsedKey: string | undefined): Promise<TPick[]> {
+export async function createQuickPickItems<TPick extends types.IAzureQuickPickItem<unknown>>(picks: TPick[] | Promise<TPick[]>, options: types.IAzureQuickPickOptions, groups: QuickPickGroup[], recentlyUsedKey: string | undefined, globalState?: Memento): Promise<TPick[]> {
```

Comment:
Maybe worth a comment to note that this is exported only for the benefit of tests. Perhaps the same for the `globalState` param.

Reply:
done